react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/RNPaywallsPackage.kt

```diff
@@ -0,0 +1,17 @@
+package com.revenuecat.purchases.react.ui
+
+import com.facebook.react.ReactPackage
+import com.facebook.react.bridge.NativeModule
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.uimanager.ViewManager
+
+
+class RNPaywallsPackage : ReactPackage {
```

Comment:
I guess this new package can also be autolinked right? (Was checking the `MainApplication` and seeing that it wasn't updated, so I'm guessing it automatically detects this package.

Reply:
The package gets added to the `getPackages` function in `PackageList.java`, which is called in `MainApplication`

```
public ArrayList<ReactPackage> getPackages() {
    return new ArrayList<>(Arrays.<ReactPackage>asList(
      new MainReactPackage(mConfig),
      new RNPurchasesPackage(),
      new RNPaywallsPackage(),
      new SafeAreaContextPackage(),
      new RNScreensPackage()
    ));
  }
```

```
private final ReactNativeHost mReactNativeHost =
      new DefaultReactNativeHost(this) {
        @Override
        public boolean getUseDeveloperSupport() {
          return BuildConfig.DEBUG;
        }

        @Override
        protected List<ReactPackage> getPackages() {
          @SuppressWarnings("UnnecessaryLocalVariable")
          List<ReactPackage> packages = new PackageList(this).getPackages();
          // Packages that cannot be autolinked yet can be added manually here, for example:
          // packages.add(new MyReactNativePackage());
          return packages;
        }
```